src/intl/bg/page-wallets.json

```
@@ -57,11 +57,11 @@
   "page-wallets-types-desc": "Има няколко начина, по които да се свържете и взаимодействате с вашия акаунт:",
   "page-wallets-web-browser": "Браузър портфейлите са уеб приложения, които ви позволяват да работите с вашия акаунт направо в браузъра",
   "page-wallets-web-browser-extension": "Браузър портфейлите с разширения са такива, които може да свалите и които ви дават възможност да работите с вашите акаунти и приложения през браузъра",
-  "page-wallets-whats-a-wallet": "Какво е портфейл в Ethereum?",
+  "page-wallets-whats-a-wallet": "Какво е портфейл в Eтереум?",
   "page-wallets-your-ethereum-account": "Вашият акаунт в Ethereum",
   "page-wallets-your-ethereum-account-desc": "Вашият портфейл е вашият прозорец към акаунта ви в Ethereum – баланс, история на трансакциите и други. Вие обаче може да смените доставчиците на портфейли по всяко време.",
   "page-wallets-your-login": "Вашата регистрация за приложенията в Ethereum",
-  "page-wallets-your-login-desc": "Вашият портфейл ви позволява да се свържете с което и да е децентрализирано приложение, като ползвате акаунта си в Ethereum. Това е като регистрация, която може да ползвате в много dapps.",
-  "additional-reading-how-to-create-an-ethereum-account": "Как да „регистрирате“ акаунт в Етереум",
+  "page-wallets-your-login-desc": "Портфейлът ви позволява да се свързвате с приложения като използвате акаунта си в Етереум. Това е подобно на регистрация, която можете да използвате в различни приложения.",
+  "additional-reading-how-to-create-an-ethereum-account": "How to create an Ethereum account",
```

Previously translation was for `How to "register" an Ethereum account`... we should just keep that until updated over reverting to English.
```suggestion
  "additional-reading-how-to-create-an-ethereum-account": "Как да „регистрирате“ акаунт в Етереум",
```